lesson_01/dasiaenglish/style.css

```diff
@@ -0,0 +1,85 @@
+
+.content{
+    margin: auto;
+    width: 50%;
+}
+
+
+.line {
+    border-bottom: 1px solid #d1d9e0b3;
+}
+
+h2, h1, h3{
+    margin: 0;
+    font-weight: 550;
+}
+
+#image {
+    margin-top: 16px;
+
+}
+
+h2 {
+    margin-top: 20px;
+}
+/*
+h1, h2 {
```

Comment:
See previous note on code that isn't being used. Please delete lines 25-51 with the commented out css.